locales/en-US/experiments.ftl

```diff
@@ -60,6 +60,29 @@ containersContributors6Title = Front End Security
 containersContributors7Title = Firefox UX
 containersContributors8Title = Firefox UX
 containersContributors9Title = Firefox UX
+firefoxlockboxDescription = Take your passwords everywhere with Firefox Lockbox, an iOS mobile app.
+firefoxlockboxIntroduction = <p>Firefox Lockbox lets you easily access your passwords everywhere. Logins are stored and synced securely, using 256-bit encryption, keeping your information safe. Easily copy username and password details to get into apps and websites. Never again find yourself locked out of your account while on your phone!</p> <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously saved logins. If you do not have an account set up, get started <a href="https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox">here</a>.</strong></p>
```

Comment:
HTML markup shouldn't really be in the string. Can this be split up in paragraphs?

Also the actual URL shouldn't be in the localized string.

Reply:
Sadly, I think we have prior art in excessive HTML markup in introductions in [at least]: min-vid, pageshot, and send:

https://github.com/mozilla/testpilot/blob/a323fac87aeee2591acb24b354906a5fb276d743/locales/en-US/experiments.ftl#L64

https://github.com/mozilla/testpilot/blob/a323fac87aeee2591acb24b354906a5fb276d743/locales/en-US/experiments.ftl#L114

https://github.com/mozilla/testpilot/blob/a323fac87aeee2591acb24b354906a5fb276d743/locales/en-US/experiments.ftl#L130

Reply:
>HTML markup shouldn't really be in the string

Ah yes, this appears to be the result of the l10n extraction. I'd have to find a clever way to redo the YAML so the "introduction" could handle these strings and still be formatted as expected and that's probably beyond my ability and scope of this PR.